sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -1,23 +1,31 @@
 <template>
-  <student-page-container>
+  <student-page-container
+    container-class="student-start-new-application"
+    :full-width="true"
+  >
+    <template #header>
+      <header-navigator
+        title="Applications"
+        subTitle="Financial Aid Application"
+      >
+      </header-navigator>
+    </template>
     <body-header>
       <template #actions>
         <v-row class="m-0 p-0 float-right">
           <v-btn
             color="primary"
-            v-if="!notDraft"
-            v-show="!isFirstPage && !submittingApplication"
+            v-if="!notDraft && !isFirstPage && !submittingApplication"
             variant="outlined"
             :loading="savingDraft"
             @click="saveDraft()"
           >
             {{ savingDraft ? "Saving..." : "Save draft" }}</v-btn
           >
           <v-btn
-            v-if="!isReadOnly"
+            v-if="!isReadOnly && !isFirstPage"
```

Comment:
also as part of refactor, we can remove `<v-progress-circular` at line 32 and use `loading` property of the btn